### DISCUSSION TITLE
Basic App Installation Error

### DISCUSSION THREAD

**Discussion by vin-ce** [vercel/next.js#21911] [Help] [answered]
Hello,
I'm trying to do the basic Next tutorial and I'm running into this error (with all prior commands and the final recommended bare bones one as seen in the screenshot).

![image](https://user-images.githubusercontent.com/57646932/107105375-f29cac80-67f3-11eb-930f-6dd0511b1e8a.png)

**Comment by michrome** [comment, marked as answer]
Hey @vin-ce try this, which is from [the Create a Next.js App article](https://nextjs.org/learn/basics/create-nextjs-app/setup):

> If you are on Windows, we recommend [downloading Git for Windows](https://gitforwindows.org/) and use Git Bash that comes with it, which supports the UNIX-specific commands in this tutorial. [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install-win10) is another option.

Then create a new directory inside `C:\Users\Vincent` and [`cd`](https://en.wikipedia.org/wiki/Cd_(command)) into it **before** running:

```
npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn-starter/tree/master/learn-starter"
```

**Reply by vin-ce** [reply]
Thank you for the reply!

Here's the error:
![image](https://user-images.githubusercontent.com/57646932/107129399-30044700-6893-11eb-8423-e440a6b02b75.png)

Here's the debug log:
[Uploading 2021-02-06T20_51_09_304Z-debug.log…]()

Does the whole issue have something to do with the fact of the space between 'Vincent' and 'Li'? 
I tried to install it in the primary 'Vincent Li' folder and got the same error.

**Reply by michrome** [reply]
I think the space is a likely culprit. I don't use Windows but adapting [this advice](https://github.com/facebook/create-react-app/issues/6512#issuecomment-625348400) you could try running:

```
npm config set cache "C:\Users\Vincen~1\AppData\Roaming\npm-cache" --global
```

Then trying the following again:

```
npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn-starter/tree/master/learn-starter"
```

As before, run these from inside file directory you created for your project.

**OR**

Create a new Windows user account without a space in the name.

**Reply by vin-ce** [reply]
That was the issue, and the npm config line fixed it - thank you!

**Reply by michrome** [reply]
Super!
